paasta_tools/autoscaling/autoscaling_service_lib.py

```diff
@@ -846,10 +847,12 @@ def autoscaling_is_paused():
         return False
 
 
-def is_autoscaling_resumed(formatted_application: Union[V1Deployment, V1StatefulSet]):
-    if formatted_application.metadata.annotations:
-        if "is_paused" in formatted_application.metadata.annotations:
-            return formatted_application.metadata.annotations["is_paused"] == "False"
+def is_autoscaling_resumed(service_config: KubernetesDeploymentConfig):
+    kube_client = KubeClient()
+    annotations = get_annotations_for_deployment(kube_client, service_config)
+    if annotations:
```

Comment:
This can just be 

    return annotations.get('paused', False) == False

right?

Also, do we have to have this "False" as a string here?

Can it just be the fact that this *has* the annotation or not?

Reply:
Yeah, "False" is going to be string when reading from k8s annotations. Also, I thought about having the annotation itself as an identifier of whether not it is paused, but for the sake of my own testing, it was better for me to explicitly see that the deployment has been paused/resumed. Maybe other devs would like to see that too?